model/clusters_mgmt/v1/aws_machine_pool_type.model

```diff
@@ -18,4 +18,7 @@ limitations under the License.
 class AWSMachinePool {
     // Use spot instances on this machine pool to reduce cost.
     SpotMarketOptions AWSSpotMarketOptions
+
+    // Additional AWS Security Groups to be added to default worker (compute) machine pool.
+    AdditionalComputeSecurityGroupIds []String
```

Comment:
Maybe name it just security group ids as machine pools are always worker/compute